### PR TITLE
🖍 Add shadow to loading spinner

### LINKED ISSUE #25538: Add box shadow on the story video loading spinner.
When the video loading spinner is displayed against a white background, it is hard to see it. Adding a box shadow should add contrast and make it easier to see. 

Here's the one we usually use: https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/amp-story.css#L876

### PR DESCRIPTION
**Pull request by Enriqe**
Closes #25538

Demo:
https://media.giphy.com/media/lPcjzAgKILkXIgdnJc/giphy.gif

@hongwei1990

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story.css [hunk 1]
```diff
@@ -684,6 +684,7 @@ amp-story .amp-video-eq,
   animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1) !important;
   animation-iteration-count: infinite !important;
   opacity: 1 !important;
+  filter: drop-shadow(0px 0px 4px rgba(0,0,0,0.35)) !important;
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story.css]: The blur radius is very high and the result looks... blurry :))

Can you try other numbers, like `0 1px 3px 1px rgba(0, 0, 0, 0.12)` maybe?

Comment (Enriqe) [reply extensions/amp-story/1.0/amp-story.css]: Done.

**Comment by Enriqe** [comment]
Friendly ping

**Comment by Enriqe** [comment]
friendly ping 🏓 @gmajoulet

**Pull request merged**